### PR TITLE
Updated indexer graphql entities

### PR DESCRIPTION
**Pull request by broody**
Added partition id, keys, and timestamp to entity table.

### CODE REVIEW

File: crates/dojo-indexer/src/graphql/entity.rs [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 use super::server::Context;
 use super::{entity_state, entity_state_update};
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
 pub struct Entity {
     pub id: String,
     pub name: Option<String>,
+    pub partition_id: String,
+    pub partition_keys: String,
```
Comment (tarrencev) [inline crates/dojo-indexer/src/graphql/entity.rs:16]: Should this be:
```suggestion
    pub keys: Vec<String>,
```

Perhaps String should be a `FieldElement` too?

File: crates/dojo-indexer/src/graphql/entity.rs [hunk 2]
```diff
@@ -57,12 +90,75 @@ pub async fn entities(context: &Context) -> FieldResult<Vec<Entity>> {
...
+    .await?;
+
+    Ok(entities)
+}
+
+pub async fn entities_by_partition_id(
+    context: &Context,
+    partition_id: String,
+) -> FieldResult<Vec<Entity>> {
+    let mut conn = context.pool.acquire().await.unwrap();
```
Comment (tarrencev) [inline crates/dojo-indexer/src/graphql/entity.rs:114]: Is it safe to unwrap like this? Or should we do some error handling?

**Review by tarrencev — APPROVED** [review]
minor feedback otherwise lgtm

File: crates/dojo-indexer/src/graphql/entity.rs (hunk 2 above)
Comment (broody) [reply crates/dojo-indexer/src/graphql/entity.rs:114]: Ah yep, will propagate the error here

File: crates/dojo-indexer/src/graphql/entity.rs (hunk 1 above)
Comment (broody) [reply crates/dojo-indexer/src/graphql/entity.rs:16]: Yeah will update name but looks like sqlite doesn't support string arrays, I guess the indexer can just insert csv string.

I think we'll have to define custom scalar for graphql to use FieldElement? I can add that as part of another PR.

**Review by tarrencev — APPROVED** [review]

**Pull request merged**